src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -51,13 +51,13 @@ public override ExportResult Export(in Batch<Metric> batch)
             {
                 var msg = new StringBuilder($"\nExport ");
                 msg.Append(metric.Name);
-                if (!string.IsNullOrEmpty(metric.Description))
+                if (metric.Description != string.Empty)
```

Comment:
similar for all exporters? wondering if there is any perf benefit of doing this, vs `IsNullOrEmpty`